CMakeLists.txt

```diff
@@ -1006,6 +1016,11 @@ foreach(hal ${OpenCV_HAL})
     add_subdirectory(3rdparty/kleidicv)
     ocv_hal_register(KLEIDICV_HAL_LIBRARIES KLEIDICV_HAL_HEADERS KLEIDICV_HAL_INCLUDE_DIRS)
     list(APPEND OpenCV_USED_HAL "KleidiCV (ver ${KLEIDICV_HAL_VERSION})")
+  elseif(hal STREQUAL "halrvv071")
```

Comment:
Looks like a merge issue

Reply:
I fixed it, I'm rebuilding it to check if it works correctly

Reply:
It seems that fix didn't help :slightly_frowning_face: